static/admin/config.yml

```
@@ -1,7 +1,7 @@
 backend:
   name: github
   repo: lynncyrin/lynncyrin-dot-me
-  branch: main
+  branch: netlifyCMS
```

⭐⭐⭐ _(must change)_

```suggestion
  branch: netlify-cms
```

capitalization in branch names is often dangerous 🙂